src/middleware/cors/index.test.ts

```diff
@@ -34,6 +34,12 @@ describe('CORS by Middleware', () => {
 
   app.use('/api5/*', cors())
 
```

Comment:
This was added in #1129, but seems to have been lost at some point :eyes:

Reply:
Thanks for the review! :pray:

I believe the addition of the following code in #1129 is to verify that duplicate headers are not set:

```typescript
  app.use(
    '/api6/*',
    cors({
      origin: 'http://example.com',
    })
  )
  app.use(
    '/api6/*',
    cors({
      origin: 'http://example.com',
    })
  )
```

If I remove one of them, should I also remove the following test?

https://github.com/honojs/hono/blob/331166471c499e5a8f432fcb77936c3d4f699cfe/src/middleware/powered-by/index.test.ts#L21-L26

Reply:
@uki00a 

Got it! Let's leave these lines. Thank you for the explanation!